package.json

```diff
@@ -39,14 +39,16 @@
     "react-datepicker": "3.1.3",
     "react-dnd": "7.7.0",
     "react-dom": "16.13.1",
+    "react-image": "4.0.3",
     "react-file-previewer": "0.6.3",
     "react-grid-layout": "1.1.1",
-    "react-hook-form": "5.7.2",
+    "react-hook-form": "6.5.0",
```

Comment:
@acrellin, will this work overall?  You tried to upgrade last time and ran into some subtleties.

Reply:
@stefanv @profjsb no, unfortunately it looks like this introduces some failures. We'll need to update our usage in order to upgrade the the new major version. Suggest reverting this change.

Reply:
Ok, I'll revert